### ISSUE TITLE
OMXPlayer being deprecated

### ISSUE THREAD

**Issue by zeusbobcat** [alubbock/rpi-vidlooper#16]
Not sure if this qualifies as an 'issue' per se, since OMXPlayer may still be available, but I thought to ask, is there a way this can work with VLC instead?

Love the software regardless! Making my life so much easier at the moment!

**Comment by haaze57** [comment]
Hello,
already thank you for this work, it's great and it allows me to finalize projects.
But actually OMXplayer is deprecated, and especially it can't decode videos with 4k resolution. Would it be possible to use VLC instead ? That would be just great !
Thanks for everything 
Regards

**Comment by alubbock** [comment]
I agree moving to VLC would be a good idea, but it'd require substantial modifications to get it working. I don't have much time to work on this right now, but perhaps at some point in the future. Alternatively, I'd be happy to review a pull request on this, if anyone wants to give implementing it a go.

**Comment by alubbock** [comment]
For anyone wanting to use the video player - as a workaround, you can install an older version of Raspberry Pi OS (based on Debian Buster) from here, which has omxplayer available: https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2021-05-28/

AFAIK the above image won't work on a Pi 5 or later.

Note that Debian Buster is end of life, so I would avoid leaving it internet connected where possible. However, it's sufficient for situations where you need an offline player.
